perfschema/init.go

```diff
@@ -242,7 +242,7 @@ func (ps *PerfSchema) buildModel(tbName string, colNames []string, cols []column
 	ps.tables[tbName] = &model.TableInfo{
 		Name:    model.NewCIStr(tbName),
 		Charset: "utf8",
-		Collate: "utf8",
+		Collate: "utf8_general_ci",
```

Comment:
Can we use the const [here](https://github.com/pingcap/tidb/blob/master/mysql/charset.go#L559)?

Reply:
Yeah, we can do this. Let me update this change.